### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by mukhaizim**
created a homepage

### CODE REVIEW

**Comment by mukhaizim** [comment]
@mechamanny

File: index.html [hunk 1]
```diff
@@ -1 +1,85 @@
...
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Travel</title>
+</head>
+<h1>Welcome to my travel page</h1> 
+<img src="Dope.jpg" alt="Dope"/>
+
+<body>
+<p> According to <a href="http://www.worldometers.info/geography/how-many-countries-are-there-in-the-world/">worldometers</a>-which has a nice dynamic table-, there are 195 countires in the world. As of September 2016 I have been to 35 countries. I am hoping that I would get a chance to visit at least 95% of the world before I die.</p>
```
Comment (mechamanny) [inline index.html:10]: This and following tags should be tabbed in.

File: pictures.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<h1>Here are some of the pictures I took while traveling</h1>
+
+<P>This is the Notre-Dame de la Garde in Marseille, France</P>
+<img src="France.jpg" alt="France"/>
+
+
+<P>This was taken at one of my favourite islands, Capri, Italy</P>
+<img src="Capri.jpg" alt="Capri"/>
+
+<p>I know the pictures are huge rght now, but I'm still learning to code so check in again next week to see a better website</p>
```
Comment (mechamanny) [inline pictures.html:16]: Will do!

File: pictures.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Travel Pictures</title>
+</head>
+<body>
+<h1>Here are some of the pictures I took while traveling</h1>
```
Comment (mechamanny) [inline pictures.html:7]: tabbing errors again.

**Pull request merged**